dask/array/routines.py

```diff
@@ -780,6 +780,30 @@ def unique(ar, return_index=False, return_inverse=False, return_counts=False):
     return result
 
 
+def _isin_kernel(element, test_elements, assume_unique=False):
+    values = np.in1d(element.ravel(), test_elements,
+                     assume_unique=assume_unique)
+    return values.reshape(element.shape + (1,) * test_elements.ndim)
+
+
+@wraps(getattr(np, 'isin', None))
+def isin(element, test_elements, assume_unique=False, invert=False):
+    element = asarray(element)
+    test_elements = asarray(test_elements)
```

Comment:
Are we ok with this as far as other libraries (e.g. xarray) are concerned?

Reply:
I think so? `test_elements` is just a set of elements to test against.

Reply:
FWIW I'm ok with it if you are. Was just reminded of issue ( https://github.com/dask/dask/issues/2912 ).